lib/read.js

```diff
@@ -128,3 +134,28 @@ module.exports = function wrapReadPixels (
 
   return readPixels
 }
+
+function isFloatSupported (gl) {
+  var texture = gl.createTexture()
+  gl.bindTexture(gl.TEXTURE_2D, texture)
+  gl.texImage2D(gl.TEXTURE_2D, 0, gl.RGBA, 1, 1, 0, gl.RGBA, gl.FLOAT, null)
+
+  var fbo = gl.createFramebuffer()
+  gl.bindFramebuffer(gl.FRAMEBUFFER, fbo)
+  gl.framebufferTexture2D(gl.FRAMEBUFFER, gl.COLOR_ATTACHMENT0, gl.TEXTURE_2D, texture, 0)
+  gl.bindTexture(gl.TEXTURE_2D, null)
+  if (gl.checkFramebufferStatus(gl.FRAMEBUFFER) !== gl.FRAMEBUFFER_COMPLETE) return false
+
+  gl.viewport(0, 0, 1, 1)
```

Comment:
That should not affect previous render/framebuffer, should that? At the moment of this call `gl` does not have any fbo/renderbuffer attached.

Reply:
Oh, and apologies, but just curious? What does this *really* do? On platforms where it's not supported, does it currently fail silently? Throw an error? A warning? After this, what is the result?

Reply:
This uses own regl ‘check’ function to throw an error if reading floats is not supported, as discussed in the adjacent PR.

Reply:
I forgot though: does it already throw an error without regl itself detecting and throwing an error?

Reply:
https://github.com/regl-project/regl/issues/487

> instead of silent failure

never mind. got it 👍